From a84326dfe6d204b2c88baddf6ed6334def032a8e Mon Sep 17 00:00:00 2001
From: Sachin Prabhu <sprabhu@redhat.com>
Date: Mon, 28 Jul 2014 12:16:41 -0400
Subject: [fs] cifs: ->rename() without ->lookup() makes no sense

Message-id: <1406549822-10376-49-git-send-email-sprabhu@redhat.com>
Patchwork-id: 86779
O-Subject: [RHEL7.1 PATCH 48/69] [bz1082049]cifs: ->rename() without ->lookup() makes no sense
Bugzilla: 1082049
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Al Viro <viro@zeniv.linux.org.uk>

Signed-off-by: Al Viro <viro@zeniv.linux.org.uk>
(cherry picked from commit 81c5a68478be38816bb5110ae0a5de1320cd2dfd)
Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/cifs/cifsfs.c b/fs/cifs/cifsfs.c
index 2411829..c3b4e13 100644
--- a/fs/cifs/cifsfs.c
+++ b/fs/cifs/cifsfs.c
@@ -861,7 +861,6 @@ const struct inode_operations cifs_file_inode_ops = {
 /* revalidate:cifs_revalidate, */
  .setattr = cifs_setattr,
  .getattr = cifs_getattr, /* do we need this anymore? */
- .rename = cifs_rename,
  .permission = cifs_permission,
 #ifdef CONFIG_CIFS_XATTR
  .setxattr = cifs_setxattr,
-- 
1.7.1